### ISSUE TITLE
Make Timed Assets/Playlist Admin process better

### ISSUE THREAD

**Issue by eosrei** [roundware/roundware-server#225]
This issues follows: https://github.com/roundware/roundware-server/issues/187

Timed Assets/Playlist functionality exists, but the admin UX isn't ideal. The TimedAsset admin form lists all assets available. Is help text needed?

How to make it better?

**Comment by hburgund** [comment]
Yes, the UX is functional, but pretty unfriendly as it stands now.  I will need to think more about the extent of ways to make it better, but a few things pop to mind immediately:

* the project selection should filter the available assets so we only have to choose from assets for the project we are working on
* only display assets of `mediatype = audio`
* we need to be able to know which asset we are selecting; the `asset id` and lat/lon isn't overly helpful
 * include ability to play the assets before selecting one (this would be ideal, but I can't visualize a good UX for this right now)
 * include other identifying asset data; `created_date`?
* It would be extremely helpful to have the ability to bulk add these (like we've done with tags - http://localhost/rw/tags/batch_add).
 * choose a project
 * add assets and time ranges line by line as many as you want

These updates would be great for the direct Timed Assets additions, but I think it will be equally important to be able to make an asset a timed playlist asset right from the asset detail admin screen as I suggested in the original issue.  Something like the way Votes are handled right now would be great.  I believe they are an inline view on the asset detail screen. No need to select project or asset, just the time ranges and BAM, done!
